homeassistant/util/yaml.py

```diff
@@ -10,6 +10,19 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class ConfigDict(OrderedDict):
```

Comment:
Yes we could indeed. How did I not see that.

On April 5, 2016 11:30:06 PM EDT, Paulus Schoutsen <notifications@github.com> wrote:
>> @@ -10,6 +10,19 @@
>>  _LOGGER = logging.getLogger(__name__)
>>  
>>  
>> +class ConfigDict(OrderedDict):
>
>We could move all this init logic into `_ordered_dict` and we don't
>even have to subclass it.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/balloob/home-assistant/pull/1718/files/898e116cfb4e79dabb4f2f3a41ed3fca6bce95b8#r58646987